plugins/vpp/ifplugin/vppcalls/vpp2101/bond_vppcalls.go

```diff
@@ -0,0 +1,118 @@
+//  Copyright (c) 2019 Cisco and/or its affiliates.
```

Comment:
What is a guideline for a copyright year? Should it be set to the current year only for new files or for updated files too?

Reply:
If a file is only copied/renamed, like in this case, then I would leave the copyright header as is.